### PR TITLE
Minor performance improvements

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: SQLitePCL.pretty/ThrowHelper.cs [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+
+namespace SQLitePCL.pretty
+{
+    /// <summary>
+    /// Provides methods for throwing common exceptions.
```
Comment (JustAMan) [inline SQLitePCL.pretty/ThrowHelper.cs:15]: it should have an explanation on _why_ it's needed

**Review by JustAMan — CHANGES_REQUESTED** [review]
I wonder if we should just upstream that...

**Comment by Bond-009** [comment]
Upstream isn’t maintained

File: SQLitePCL.pretty/ThrowHelper.cs (hunk 1 above)
Comment (Bond-009) [reply SQLitePCL.pretty/ThrowHelper.cs:15]: I’m on my phone, so I can’t edit it, but here is the explanation: https://reubenbond.github.io/posts/dotnet-perf-tuning

Comment (JustAMan) [reply SQLitePCL.pretty/ThrowHelper.cs:15]: Can you add it when you're back at laptop?

**Review by JustAMan — APPROVED** [review]

**Pull request merged**